scss/_mixins.scss

```diff
@@ -65,58 +65,21 @@
 	padding: 0;
 }
 
-@mixin nListsItem($use-case: n-lists) {
-	@include oColorsFor($use-case, background);
-	border: 1px solid rgba(oColorsGetColorFor($use-case, border), 0.25);
-	border-width: 0 0 1px;
-	min-width: 200px;
-	vertical-align: top;
-
-	> a {
-		@include nListsItemLink($use-case);
-	}
-}
-
-@mixin nListsItemLink($use-case: n-lists) {
-	@include oTypographyTopic;
-	display: block;
-	padding: 2px 15px 2px $spacing-unit/2;
-	position: relative;
-	text-decoration: none;
-	border-bottom-width: 0;
-	&:after {
-		@include oIconsGetIcon('arrow-right', oColorsGetColorFor($use-case, border), 10);
-		content: '';
-		opacity: 0.25;
-		position: absolute;
-		right: 0;
-		top: 8px;
-	}
-	&:hover,
-	&:focus {
-		background: rgba(oColorsGetColorFor(#{$use-case}-hover, background), 0.1);
-		border-bottom-width: 0;
-		&:after {
-			@include oIconsGetIcon('arrow-right', oColorsGetColorFor(#{$use-case}-hover, border), 10);
-		}
-	}
-}
-
```

Comment:
was this never used?

Reply:
Nope https://github.com/search?q=org%3AFinancial-Times+nListsItem&type=Code

Reply:
nice, love too delete the unused mixin